packages/gestalt/src/InternalLink.js

```diff
@@ -104,15 +104,15 @@ const InternalLinkWithForwardRef: AbstractComponent<Props, HTMLAnchorElement> =
     linkStyles.link,
     focusStyles.hideOutline,
     touchableStyles.tapTransition,
-    inline ? linkStyles.inlineBlock : linkStyles.block,
     getRoundingClassName(isTapArea ? rounding || 0 : 'pill'),
     {
       [touchableStyles.tapCompress]: !disabled && tapStyle === 'compress' && isTapping,
       [focusStyles.accessibilityOutline]: !disabled && isFocusVisible,
     },
     isButton
       ? {
-          [linkStyles.buttonLink]: true,
+          [linkStyles.inlineFlex]: inline,
```

Comment:
These will now be flex? Will that affect current usages?

Reply:
![image](https://user-images.githubusercontent.com/10593890/116450894-d83e0200-a829-11eb-9486-51788d26eb01.png)


It was already flex 
![image](https://user-images.githubusercontent.com/10593890/116450932-e55af100-a829-11eb-8361-0209f7f3caeb.png)

Reply:
We are just adding the inline to fix that issue.

Reply:
![image](https://user-images.githubusercontent.com/10593890/116452832-28b65f00-a82c-11eb-8e63-c6aea93ac196.png)
 >>